Paccy10/olympus-api
# Fetaure(bookings): fetch single booking

**What does this PR do?**

- Fetch single booking

**Description of Task to be completed?**

- Any user can fetch his own booking or an admin can fetch any user booking by id

**How should this be manually tested?**

- Checkout to the branch `ft-fetch-single-booking`
- Run the app with `flask run`
- Send a `GET` request to /bookings/{booking_id} and replace booking_id with the booking ID
- The booking with the given ID should be fetched

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
